src/app/components/weather-in-seven-days/weather-in-seven-days.component.html

```diff
@@ -0,0 +1,12 @@
+<div>
+  <h1>London</h1>
+  <div class="days-of-the-week-container">
+    <h2 class="day-of-the-week">Monday</h2>
+    <h2>Tuesday</h2>
```

Comment:
I was really expecting something like

<app-forecast-component [date]="mondayDate" forecast="sunny" />
<app-forecast-component [date]="tuesdayDate" forecast="sunny" />

or even better

<app-forecast-component *ngFor="let dailyForecast of week" [date]="dailyForecast.date" forecast="dailyForecast.weather" />